src/Core/Framework/Adapter/Cache/CacheInvalidator.php

```diff
@@ -100,4 +101,10 @@ private function purge(array $keys): void
 
         $this->dispatcher->dispatch(new InvalidateCacheEvent($keys));
     }
+
+    private function shouldForceInvalidate(): bool
+    {
+        return $this->environment !== 'prod'
```

Comment:
why do we need the environment check. on dev anyway the cache is mostly disabled 🤔

Reply:
That check was there before: https://github.com/shopware/shopware/pull/6061/files#diff-ae481c26f2431679c742b89d5c458b8ba8762f864508c65fa13f5e5f3c30ad38L46

Personally i agree that we can remove it, probably would make debugging etc easier

@OliverSkroblin do you have any opinion on this? do we miss something? as you originally added that check

Reply:
I'll remove the check for now, if it served a purpose we'll bring it back